app/controllers/catalog_controller.rb

```diff
@@ -8,8 +8,7 @@ class CatalogController < ApplicationController
   include Hydra::Catalog
   include Hydra::Controller::ControllerBehavior
   include ConvertIds
-  # These before_filters apply the hydra access controls
-  # before_filter :enforce_show_permissions, :only=>:show
```

Comment:
Ok, for this PR, I'll just put it back the way it was, and then I'll send a separate PR for uncommenting it.